scripts/ci-e2e.sh

```diff
@@ -110,7 +113,7 @@ init_networks() {
       --allow tcp:443 --network "${GCP_NETWORK_NAME}" --quiet
     gcloud compute firewall-rules create "${GCP_NETWORK_NAME}"-allow-icmp --project "$GCP_PROJECT" \
       --allow icmp --network "${GCP_NETWORK_NAME}" --priority 65534 --quiet
-    gcloud compute firewall-rules create "${GCP_NETWORK_NAME}"-allow-internal --project "$GCP_PROJECT" \
+    gcloud compute firewall-rules create "${NETWGCP_NETWORK_NAMEORK}"-allow-internal --project "$GCP_PROJECT" \
```

Comment:
Is this a typo?

Reply:
it is yes. This is actually still a WIP. I wanted to separate the work here so that the main work can be merged. similar to other features.